src/reactor.h

```diff
@@ -0,0 +1,242 @@
+  #pragma cyclus var { \
+    "doc": "Fresh fuel commodity", \
+    "tooltip": "Name of fuel commodity requested", \
+    "uilabel": "Fuel input commodity" \
+  }
+  std::string blanket_incommod;
+
+  #pragma cyclus var { \
+    "doc": "Fresh fuel commodity", \
+    "tooltip": "Name of fuel commodity requested", \
+    "uilabel": "Fuel input commodity" \
+  }
+  std::string blanket_inrecipe;
+
+  #pragma cyclus var { \
+    "default": 0.3, \
+    "doc": "Li-6 enrichment requirement percent as a decimal", \
+    "tooltip": "Defaults to 30% (0.3)", \
+    "units": "%", \
+    "uilabel": "Minimum Blanket Enrichment" \
+  }
+  double min_blanket_enrichment;
+
+
+  //WARNING: The default on this is completely arbitrary!
+  #pragma cyclus var { \
+    "default": 0.05, \
+    "doc": "Percent of blanket that gets recycled every timestep", \
+    "tooltip": "Defaults to 5% (0.05), must be between 0 and 15%", \
+    "units": "%", \
+    "uilabel": "Blanket Turnover Rate" \
+  }
+  double blanket_turnover_rate;
+
+  bool operational = true; 
+  bool core_loaded = false;
+
+  #pragma cyclus var {"tooltip":"Buffer for handling tritium kept in the core"}
+  cyclus::toolkit::ResBuf<cyclus::Material> tritium_core;
+
+  #pragma cyclus var {"tooltip":"Buffer for handling tritium kept in reserve"}
+  cyclus::toolkit::ResBuf<cyclus::Material> tritium_reserve;
+
+  #pragma cyclus var {"tooltip":"Buffer for handling excess tritium material to be sold"}
+  cyclus::toolkit::ResBuf<cyclus::Material> tritium_storage;
+
+  #pragma cyclus var {"tooltip":"Buffer for handling helium-3 byproduct material"}
+  cyclus::toolkit::ResBuf<cyclus::Material> helium_storage;
+
+  #pragma cyclus var {"tooltip":"Buffer for handling enriched lithium blanket material"}
+  cyclus::toolkit::ResBuf<cyclus::Material> blanket;
+
+  #pragma cyclus var {"tooltip":"Tracker to handle on-hand tritium"}
+  cyclus::toolkit::TotalInvTracker fuel_tracker;
+
+  #pragma cyclus var {"tooltip":"Tracker to handle blanket material"}
+  cyclus::toolkit::TotalInvTracker blanket_tracker;
+  
+  #pragma cyclus var {"tooltip":"Tracker to handle excess tritium to be sold"}
+  cyclus::toolkit::TotalInvTracker excess_tritium_tracker;
+
+  #pragma cyclus var {"tooltip":"Tracker to handle on-hand helium"}
+  cyclus::toolkit::TotalInvTracker helium_tracker;
+
+  cyclus::toolkit::MatlBuyPolicy fuel_startup_policy;
+  cyclus::toolkit::MatlBuyPolicy fuel_refill_policy;
+
+  //These two may be redundant (only one may be needed)
+  //cyclus::toolkit::MatlBuyPolicy blanket_startup_policy;
+  cyclus::toolkit::MatlBuyPolicy blanket_fill_policy;
+
+  cyclus::toolkit::MatlSellPolicy tritium_sell_policy;
+  cyclus::toolkit::MatlSellPolicy helium_sell_policy;
+
+  //For the schedule buy policy
+  //cyclus::IntDistribution::Ptr active_dist = NULL;
+  //cyclus::IntDistribution::Ptr dormant_dist = NULL;
+  //cyclus::IntDistribution::Ptr size_dist = NULL;
+
+  void Startup();
+  void OperateReactor(double TBR, double burn_rate=55.8);
```

Comment:
Magic number: where does 55.8 come from?

Reply:
I added a comment with a citation and explanation about what this value is and where it comes from.